### DISCUSSION TITLE
Airflow.cfg configuration

### DISCUSSION THREAD

**Discussion by nitra-1** [apache/airflow#25733] [General] [answered]
In master-worker setup of Airflow
What should we use, celeryexecutor or localexecutor in the master / worker airflow.cfg ?

**Comment by alexkruc** [comment, marked as answer]
Airflow should use the same executor across all your components, or it can lead to wrong behaviour.
In your case, you should use `CeleryExecutor` on both your "master" (I assume it's scheduler & webserver), and in the worker nodes.

`LocalExecutor` will make the scheduler (or more accurately, executor) run tasks on the local machine, which is not what you want..

You can read more about executors here:
https://airflow.apache.org/docs/apache-airflow/stable/executor/index.html
